Distribution/Server/Features/Tags.hs

```diff
@@ -323,20 +322,22 @@ constructCategoryTags = map (tagify . map toLower) . fillMe . categorySplit . ca
 constructImmutableTags :: GenericPackageDescription -> [Tag]
 constructImmutableTags genDesc =
     let desc = flattenPackageDescription genDesc
-        !l = license desc
+--        !l = license desc
         !hl = hasLibs desc
         !he = hasExes desc
 -- These tags are too noisy and don't provide a good signal anymore
 --        !ht = hasTests desc
 --        !hb = hasBenchmarks desc
-    in licenseToTag l
+    in [] -- licenseToTag l
     ++ (if hl then [Tag "library"] else [])
     ++ (if he then [Tag "program"] else [])
 -- These tags are too noisy and don't provide a good signal anymore
 --    ++ (if ht then [Tag "test"] else [])
 --    ++ (if hb then [Tag "benchmark"] else [])
     ++ constructCategoryTags desc
   where
+{- temporarily disabled until we figure out whether mapping SPDX to tags is desirable
```

Comment:
we can map "simple identifiers" to tags, i.e .`License (ELicense (ELicenseId licId) Nothing)`